edgeless_orc/src/lib.rs

```diff
@@ -32,6 +32,10 @@ pub async fn edgeless_orc_main(settings: EdgelessOrcSettings) {
     let (mut orchestrator, orchestrator_task) = orchestrator::Orchestrator::new(settings.clone()).await;
 
     let orchestrator_server = edgeless_api::grpc_impl::orc::OrchestratorAPIServer::run(orchestrator.get_api_client(), settings.orchestrator_url);
+    let orchestrator_coap_server = edgeless_api::coap_impl::orchestration::CoapOrchestrationServer::run(
```

Comment:
I think this means that the orchestrator will _always_ run both the gRPC and the CoAP interfaces.

Does it make sense to let the user choose in `orchestrator.toml`?

Reply:
Yes, I just wasn't able to add that yet.